src/Analysis/AnalysisApp.jsx

```diff
@@ -3,13 +3,17 @@ import PropTypes from 'prop-types'; // see https://github.com/facebook/prop-type
 import Select from 'react-select';
 import { Spin } from 'antd';
 import Button from '@gen3/ui-component/dist/components/Button';
+import { ReactQueryDevtools } from 'react-query/devtools';
```

Comment:
we don't need dev tools in production codes

Reply:
removed